components/brave_rewards/resources/page/promos/index.tsx

```diff
@@ -68,6 +79,17 @@ const getRootImagePath = (path: string) => {
 
 export const getPromo = (type: PromoType, rewardsData: Rewards.State) => {
   switch (type) {
+    case 'bitflyer-verification':
+      return {
+        link: getLink(type),
+        copy: (
```

Comment:
I think this can also be `copy: getLocale(...),` now?

Reply:
Yes, fixed.